data/quarg/quarg missions.txt

```diff
@@ -325,3 +325,150 @@ mission "Quarg Pug Arfecta Warning"
+			label "heliarch ask"
+			branch end
+				and
+					has "ask quarg coalition late: threat"
+					has "ask quarg coalition late: aid"
+			choice
+				`	"Is that a threat?"`
+					goto threat
+						to display
+							not "ask quarg coalition late: threat"
+				`	"What if I do want to aid the Heliarchs?"`
+					goto aid
+						to display
+							not "ask quarg coalition late: aid"
+			label threat
+			action
+				set "ask quarg coalition late: threat"
+			`	"No. It is advice. Though you have joined them, you are still welcome at our rings, for the time being. But, the Heliarchs are a group formed and bound by their distrust and repulse for our kind. Should you continue to follow the Heliarchs, sooner or later they are sure to put you against us, at which point I can no longer guarantee you will retain safe passage through our systems, or will be granted landing clearance on our worlds still."`
+				goto "heliarch ask"
+			label aid
+			action
+				set "ask quarg coalition late: aid"
+			`	"Your motives are your own, but if captivated by own obsessions you were, then I can only hope you will come to see reason before you too are lost to their path."`
+				goto "heliarch ask"
+			label "lunarium 2"
+			`	It then turns to face you again, saying, "I am aware that you brought one from the Coalition to us, for a conversation. One belonging to a group that seeks to overthrow the Heliarchs. Do be careful, human, they might not be as technologically advanced as us Quarg, but they are very strong still, and duty-bound unlike most others. You and your allies are bound to face tremendous hardship, if you are to oppose them."`
+			label "lunarium ask"
+			branch end
+				and
+					has "ask quarg coalition late: help"
+					has "ask quarg coalition late: tips"
+			choice
+				`	"Will the Quarg help us fight the Heliarchs?"`
+					goto help
+						to display
+							not "ask quarg coalition late: help"
+				`	"Any tips on how to fight them?"`
+					goto tips
+						to display
+							not "ask quarg coalition late: tips"
+			label help
+			action
+				set "ask quarg coalition late: help"
+			`	"We agreed to that conversation, that 'interview' of yours. Beyond that, we cannot help much without provoking war, I fear. We seek not to strike fear into the hearts of those of the Coalition, we never did. It would burden your cause to have us take part in the fighting."`
+				goto "lunarium ask"
+			label tips
+			action
+				set "ask quarg coalition late: tips"
+			`	"Do not be arrogant. Do not underestimate them. Do not take victory for granted, no matter how cornered they may seem, lest cornered yourself you shall be."`
+				goto "lunarium ask"
+			label end
+			action
+				clear "ask quarg coalition late: help
+				clear "ask quarg coalition late: tips
+				clear "ask quarg coalition late: threat
+				clear "ask quarg coalition late: aid
```

Comment:
```suggestion
				clear "ask quarg coalition late: help"
				clear "ask quarg coalition late: tips"
				clear "ask quarg coalition late: threat"
				clear "ask quarg coalition late: aid"
```

Reply:
Was fixing after I saw the failed check, missed your suggestion here lol